src/components/Sing/ToolBar.vue

```diff
@@ -181,18 +191,25 @@ const setBeatTypeInputBuffer = (beatTypeStr: string | number | null) => {
 
 const playheadTicks = ref(0);
 
-const playheadPositionStr = computed(() => {
+const playheadPositionMinSecStr = computed(() => {
```

Comment:
最小の`sec`と紛らわしいのでコメントだけ！
```suggestion
/// 再生時間の分と秒
const playheadPositionMinSecStr = computed(() => {
```